extensions/cornerstone/src/getCustomizationModule.ts

```diff
@@ -64,6 +64,136 @@ function getCustomizationModule() {
             step: 1,
           },
         },
+        {
+          id: 'cornerstone.measurements',
+          Angle: {
```

Comment:
please refactor in another file, we should call this something like cornerstone.measurementStatsDisplay or something